src/Altinn.Common.PEP/Altinn.Common.PEP/Authorization/AppAccessHandler.cs

```diff
@@ -47,9 +48,13 @@ public class AppAccessHandler : AuthorizationHandler<AppAccessRequirement>
         /// <returns>A Task</returns>
         protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context, AppAccessRequirement requirement)
         {
+            HttpContext httpContext = _httpContextAccessor.HttpContext;
```

Comment:
Are not the headers availble in the AuthorizationHandlerContext? Thought is was just a variant of httpcontext